packages/wonder-blocks-form/src/components/__tests__/text-area.test.tsx

```diff
@@ -0,0 +1,1077 @@
+                        value="Text"
+                        onChange={() => {}}
+                        aria-label={ariaLabel}
+                    />,
+                    defaultOptions,
+                );
+                // Act
+
+                // Assert
+                const textArea = await screen.findByRole("textbox");
+                expect(textArea).toHaveAttribute("aria-label", ariaLabel);
+            });
+
+            it("should set the aria-labelledby attribute when provided", async () => {
+                // Arrange
+                const ariaLabelledBy = "test-label-id";
+                render(
+                    <TextArea
+                        value="Text"
+                        onChange={() => {}}
+                        aria-labelledby={ariaLabelledBy}
+                    />,
+                    defaultOptions,
+                );
+                // Act
+
+                // Assert
+                const textArea = await screen.findByRole("textbox");
+                expect(textArea).toHaveAttribute(
+                    "aria-labelledby",
+                    ariaLabelledBy,
+                );
+            });
+
+            it("should set the aria-describedby attribute when provided", async () => {
+                // Arrange
+                const ariaDescribedBy = "test-label-id";
+                render(
+                    <TextArea
+                        value="Text"
+                        onChange={() => {}}
+                        aria-describedby={ariaDescribedBy}
+                    />,
+                    defaultOptions,
+                );
+                // Act
+
+                // Assert
+                const textArea = await screen.findByRole("textbox");
+                expect(textArea).toHaveAttribute(
+                    "aria-describedby",
+                    ariaDescribedBy,
+                );
+            });
+
+            it("should set the aria-details attribute when provided", async () => {
+                // Arrange
+                const ariaDetails = "details-id";
+                render(
+                    <TextArea
+                        value="Text"
+                        onChange={() => {}}
+                        aria-details={ariaDetails}
+                    />,
+                    defaultOptions,
+                );
+                // Act
+
+                // Assert
+                const textArea = await screen.findByRole("textbox");
+                expect(textArea).toHaveAttribute(
+                    "aria-details",
+                    `${ariaDetails}`,
+                );
+            });
+
+            it("should set aria-invalid to true if the validate prop returns an error message", async () => {
+                // Arrange
+                render(
+                    <TextArea
+                        value="text"
+                        onChange={() => {}}
+                        validate={() => "Error"}
```

Comment:
**suggestion:** It took me a moment to understand why ariaInvalid is true in this test. Could you please add a comment here explaining how this causes that?